packages/dashboard/src/components/appbar.tsx

```diff
@@ -13,15 +13,17 @@ import SettingsIcon from '@material-ui/icons/Settings';
 import React from 'react';
 import { AuthenticatorContext, UserContext } from './auth/contexts';
 import HelpIcon from '@material-ui/icons/Help';
+import { EmergencyAlarm } from './emergency-alarm';
 
 export interface AppBarProps {
   toggleShowOmniPanel(): void;
   showSettings(show: boolean): void;
   showHelp(show: boolean): void;
+  alarmState: boolean | null;
```

Comment:
make it optional, with a `TODO` tag explaining it's not currently used because of the lack of usable backend.